processor/lsmintervalprocessor/processor_test.go

```diff
@@ -292,7 +293,11 @@ func TestClientMetadata(t *testing.T) {
 }
 
 func BenchmarkAggregation(b *testing.B) {
-	benchmarkAggregation(b, nil)
+	benchmarkAggregation(b, nil, false)
+}
+
+func BenchmarkAggregationParallel(b *testing.B) {
+	benchmarkAggregation(b, nil, true)
 }
```

Comment:
Do we really need two benchmarks? If you want to test without parallelism, you can run with `-cpu=1`